### ISSUE TITLE
npm install error: "npm ERR! code ERESOLVE"

### ISSUE THREAD

**Issue by Jeff-411** [aleksey-hoffman/sigma-file-manager#191]
### Is there an existing issue for this?

- [X] I have searched the opened issues

### Problem description

I'm running node v16.17.0 with npm 8.15.0.

When I run `npm install` on the cloned repo, I get a "npm ERR! code ERESOLVE" error.

The full error message in the terminal, the eresolve report, and the debug log are attached below.

[install-terminal-output.txt](https://github.com/aleksey-hoffman/sigma-file-manager/files/9499998/install-terminal-output.txt)

[eresolve-report.txt](https://github.com/aleksey-hoffman/sigma-file-manager/files/9500000/eresolve-report.txt)

[2022-09-05T16_39_30_631Z-debug-0.log](https://github.com/aleksey-hoffman/sigma-file-manager/files/9500003/2022-09-05T16_39_30_631Z-debug-0.log)

### Steps to reproduce

1. run `npm install`

### Environment

- OS: Windows 10
- App version: 1.5 (latest)

**Comment by aleksey-hoffman** [comment]
Yeah mate, it can only run on Node v15.14.0 at the moment. You can find the download link here
https://nodejs.org/en/download/releases/

Or, here's the direct link for Windows installable for ya:
https://nodejs.org/download/release/v15.14.0/node-v15.14.0-x64.msi

**Comment by aleksey-hoffman** [comment]
It'd be even better to use [NVM](https://github.com/coreybutler/nvm-windows). It allows you to install multiple Node versions on your system and then switch between them easily
